src/components/MealCard.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import '../css/components/MealResultsListing.css';
+
+class MealCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {   
+    };
+  }
+
+  render() {
+    return (
+
+        <div className="singleColumn">
```

Comment:
We should try to keep class names consistent. We started off with having this sort of naming: `this-is-a-class`. It would probably be good to have it in the rest of our code.